bin/check-swap-percent.rb

```diff
@@ -44,6 +44,12 @@ def run
     total_swap = output_swap.split(' ')[1].to_i
     used_swap = output_swap.split(' ')[2].to_i
 
+    # Check if swap exists
+    if total_swap.zero?
+      print 'There is no SWAP on this machine'
+      exit
```

Comment:
I agree it might be useful to provide it optionally. We could do something like this:
`    
# Check if swap exists
if total_swap.zero?
  if config[:required]
    critical 'ERROR: SWAP is missing on this machine'
  else
    print 'There is no SWAP on this machine'
    exit
  end
end
`  
Should I update the pull request?